chain/client/src/stateless_validation/chunk_validator/orphan_witness_handling.rs

```diff
@@ -117,6 +117,8 @@ impl Client {
             match self.chain.get_block_header(new_block.header().last_final_block()) {
                 Ok(block_header) => block_header,
                 Err(err) => {
+                    // TODO(wacban) this error happens often in integration
+                    // tests when the last final block is genesis / genesis.prev.
```

Comment:
Has this something to do with the new congestion control fields. or is it just an on-the-fly observation?